tests/test_system_stats.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from jobrunner.system_stats import get_system_stats
+
+
+@pytest.mark.needs_docker
+def test_get_system_stats():
+    stats = get_system_stats()
+    assert {
+        "total_disk_space",
+        "available_disk_space",
+        "total_memory",
+        "available_memory",
+    } <= stats.keys()
```

Comment:
TIL you can use order comparison on sets/dict views! This is nice, but was also very confusing! But I don't know a nicer way of expressing that, since dict views are annoyingly not quite sets